ietf-optical-impairment-topology.tree

```diff
@@ -183,10 +187,10 @@ module: ietf-optical-impairment-topology
                    +--ro loss    decimal64
   augment /nw:networks/nw:network/nw:node/tet:te
             /tet:tunnel-termination-point:
-    +--ro transceiver* []
-       +--ro transponder-ref?
+    +--ro transceiver* [transponder-ref transceiver-ref]
```

Comment:
Does this list need to be keyed with transponder-ref+transceiver-ref, or not?

Reply:
This is the piece of YANG tree have been presented by Italo and agreed during discussion in the weekly:

+--rw tunnel-termination-point* [tunnel-tp-id]
   +--rw tunnel-tp-id                           binary

OLD
   +--ro transceiver* []
NEW
   +--ro transceiver* [transponder-ref, tranceiver-ref]

   |  +--ro transponder-ref?
   |  |       -> ../../../../transponder/transponder-id
   |  +--ro tranceiver-ref?    leafref